src/Main.java

```diff
@@ -0,0 +1,585 @@
+                    System.out.println(" - Can be size Small, Medium, or Large");
+                    System.out.println("Price: Small = $1.99, Medium = $2.99, Large = $3.99");
+
+                    // Asks to add to cart
+                    while(going == true){
+                        System.out.println("Would you like to this add item your order? (Y or N)");
+                        addToCart = readCharacter();
+                        
+                        if(addToCart == 'Y'){
+
+                            // What the user wants
+                            while(going == true){
+                                System.out.println("What size would you like (Small or Medium or Large)");
+                                size = readLine();
+                                if(size.equals("Small")) break;
+                                else if (size.equals("Medium")) break;
+                                else if(size.equals("Large")) break;
+                                else error();
+                            }
+
+                            // Asks for name and creates the object and adds it to the order
+                            System.out.println("Name for Root Beer: ");
+                            Drink rootBeer = new Drink(readLine(), size);
+                            order.addFoodItem(rootBeer);   
+                            //foodType.add("Root Beer");
+                            break;
+
+                        } else if(addToCart == 'N'){
+                            break;
+                        } else error();
+                            
+                    }
+                }
+
+        } else error();
+        
+        // Ask if they want to proceed to checkout or still order
+        while(going  == true){
+            System.out.println("Would you like to keep ordering / browsing? (Y or N)");
+            yOrN = readCharacter();
+            if (yOrN == 'Y'){
+                break;
+            } else if (yOrN == 'N'){
+                going = false;
+                break;
+            } else error();
+        } 
+    }
+
+    ArrayList<Food> foodList = order.getFoodList(); 
+
+    // Output all items, their names, thier prices, then subtotal, tax, and total.
+    for(int i = 0; i < foodList.size(); i++){
+        System.out.println("---------");
+        /*if(foodType.get(i).equals("MorningBurger")){
+            System.out.println(foodList.get(i).getBagelBun()? "Yes":"No");
+        }  */
+        System.out.println(foodList.get(i));
+        System.out.println(foodList.get(i).getName());
+        System.out.println("$ " + foodList.get(i).getPrice());
+    }
+    System.out.println("----------");
+    System.out.println("Sub-Total: $" + String.format("%.2f",order.getSubTotal()));
+    System.out.println("Tax: $" + String.format("%.2f",order.getTax()));
+    System.out.println("Total: $" + String.format("%.2f",order.getTotalPrice()));
+    }
```

Comment:
Order printout functionality should be encapsulated in the Order class and not implemented in the Main program